src/OrbitQt/orbitaboutdialog.ui

```diff
@@ -77,7 +70,36 @@
      </widget>
     </widget>
    </item>
-   <item row="3" column="0">
+   <item row="6" column="0">
+    <widget class="QDialogButtonBox" name="buttonBox">
+     <property name="orientation">
+      <enum>Qt::Horizontal</enum>
+     </property>
+     <property name="standardButtons">
+      <set>QDialogButtonBox::Close</set>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="0">
+    <widget class="QLabel" name="titleLanel">
```

Comment:
titleLanel -> titleLabel

Reply:
Since this has not been touched by me I will leave it as-is. Thanks anyway!